controllers/rcj_soccer_referee_supervisor/referee/referee.py

```diff
@@ -1,44 +1,248 @@
 import random
-from typing import Optional
+import struct
+from typing import List, Optional, Tuple
+
+from controller import Supervisor
 
 from referee.consts import (
+    BALL_INITIAL_TRANSLATION,
```

Comment:
@Adman I guess `black` has not been applied here yet, I have hard time it accepting this ALL_UPPER/CammelCase variables :smile:

Reply:
isort has been applied here sorting imports alphabetically.